arm-web/2016-03-01/ResourceProvider.json

```diff
@@ -207,76 +271,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/publishingCredentials": {
```

Comment:
@salameer @ravbhatnagar - what is the ARM guideline for removing an API from an existing api-version ?